tests/scaletest/lib/ingress.go

```diff
@@ -333,15 +349,49 @@ func DeleteIngress(namespace string, listOfIngressToDelete []string) ([]string,
 	return listOfDeletedIngresses, nil
 }
 
-func ListIngress(t *testing.T, namespace string) error {
-	t.Logf("Listing ingress in namespace %q:\n", namespace)
+func UpdateIngress(namespace string, listOfIngressToUpdate []string) ([]string, error) {
+	for i := 0; i < len(listOfIngressToUpdate); i++ {
+		retryErr := retry.RetryOnConflict(retry.DefaultRetry, func() error {
+			result, getErr := kubeClient.Resource(ingressResource).Namespace(namespace).Get(ctx, listOfIngressToUpdate[i], metaV1.GetOptions{})
```

Comment:
Since you are referencing the `listOfIngressToUpdate[i]` anyway, can we just change the for loop above to:
```
for _, ing := range listOfIngressToUpdate {
```

Reply:
Done!